README.md

```diff
@@ -57,25 +63,35 @@ jobs:
   clang-tidy:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
-      - name: Build project and/or unit tests
-        run: ./your-build-script.sh
-      - name: Run clang-tidy
-        run: ./your-clang-tidy-script.sh --fixes-path fixes.yaml
-      - name: Run clang-tidy-pr-comments action
-        uses: platisd/clang-tidy-pr-comments@master
-        with:
-          # The GitHub token (or a personal access token)
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          # The path to the clang-tidy fixes generated previously
-          clang_tidy_fixes: fixes.yaml
-          # Optionally set to true if you want the Action to request
-          # changes in case warnings are found
-          request_changes: true
-          # Optionally set the number of comments per review
-          # to avoid GitHub API timeouts for heavily loaded
-          # pull requests
-          suggestions_per_comment: 10
+    - uses: actions/checkout@v2
+    - name: Install clang-tidy
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y clang-tidy
+    - name: Prepare compile_commands.json
+      run: |
+        cmake -B build -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
```

Comment:
Is it necessary to export the compile commands?
I've been [using it without](https://github.com/platisd/smartcar_shield/blob/master/.github/workflows/tests.yml).

Reply:
I don't use the `run-clang-tidy`, but at least `clang-tidy-diff` wants to have the `compile_commands.json` to know the build settings (defines, include paths and so on).

Reply:
OK I see, haven't used it myself. Can we document that `compile_commands.json` is necessary for `clang-tidy-diff` then and mention somewhere that `run-clang-tidy` is also an option? I am/was under the impression the latter is the go-to choice for running clang-tidy generally.

Reply:
I can offer the `run-clang-tidy` first before offering the `clang-tidy-diff`, but I don't have a working example, since I don't use it myself.

Reply:
Don't worry about a working example, just mention it in the text so the readers can google it.